### PR TITLE
[Merged by Bors] - Add support for Black Hole connector

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

## Review Checklist

- [x] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/usage_guide/pages/catalogs/tpcds.adoc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 = TPC-DS
 
-Trino provides connectors that provide a data source for the TCP-DS benchmark data set. This is not a stored data set and is instead a virtual data sets generated at query time using a deterministic algorithm.
+The https://trino.io/docs/current/connector/tpcds.html[TPC-DS connector] provides a data source for the TCP-DS benchmark data set.
+This is not a stored data set and is instead a virtual data sets generated at query time using a deterministic algorithm.
```
Comment (maltesander) [inline docs/modules/usage_guide/pages/catalogs/tpcds.adoc:4]:
```suggestion
This is not a stored data set and is instead a virtual data set generated at query time using a deterministic algorithm.
```

File: docs/modules/usage_guide/pages/catalogs/tpch.adoc [hunk 2]
```diff
@@ -1,6 +1,7 @@
 = TPC-H
 
-Trino provides connectors that provide a data source for the TCP-H benchmark data set. This is not a stored data set and is instead a virtual data sets generated at query time using a deterministic algorithm.
+The https://trino.io/docs/current/connector/tpch.html[TPC-H connector] provides a data source for the TCP-H benchmark data set.
+This is not a stored data set and is instead a virtual data sets generated at query time using a deterministic algorithm.
```
Comment (maltesander) [inline docs/modules/usage_guide/pages/catalogs/tpch.adoc:4]:
```suggestion
This is not a stored data set and is instead a virtual data set generated at query time using a deterministic algorithm.
```

**Review by maltesander — COMMENTED** [review]
LGTM overall.
Should we mention in the docs that this does not work for HA:
`This connector does not work properly with multiple coordinators, since each coordinator has different metadata.` ?

**Comment by sbernauer** [comment]
Honestly i think it's fine as Trino doesn't support multiple coordinators as of now

**Comment by maltesander** [comment]
> Honestly i think it's fine as Trino doesn't support multiple coordinators as of now

Yeah but that may change in the future?

**Comment by sbernauer** [comment]
But i that case the blackhole connector might supports it as well ;)
The more we put in our docs rather than referencing to the official docs the more might be outdated

Mentioned it anyway ;)

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by sbernauer** [comment]
thx!

**Pull request closed without merging**